src/Executable.js

```diff
@@ -153,6 +162,19 @@ export default class Executable {
         }
     }
 
+    /**
+     * Get the list of node account IDs set in the transaction. If no nodes are set, then null is returned.
+     *
+     * @returns {?string[]}
+     */
+    get transactionNodeAccountIds() {
+        if (this._transactionNodeIds.isEmpty) {
+            return null;
```

Comment:
Might be better to have it return [] so you wouldn't have to worry about types like `undefined`, `null`, etc.

Reply:
All the other getters return `null` when empty, which is preferable because it eliminates the need to check the length when adding conditions or verifying existence.